src/Ren/Compiler/Emit/ESModule.elm

```diff
@@ -74,6 +74,14 @@ import_ { path, name, exposed } =
                 ]
 
 
+import_specifier : Module.ImportSpecifier -> Pretty.Doc t
```

Comment:
Minor nit-pick, by Elm identifiers are typically camelCase rather than snake_case. If you noticed `import_` defined elsewhere, that's because `import` is a keyword - it's kinda ugly but it'll have to do.

Reply:
Yeah, I figured that was the logic behind `import_`, but seeing it did still flip the `snake_case` switch in my brain 😂